.github/workflows/run_doublet-detection.yml

```diff
@@ -39,14 +40,21 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Download test data in SCE format
+        env:  |
```

Comment:
No pipe needed here
```suggestion
        env: 
```

Reply:
beat you https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/638/commits/43954b245421795881671f37b065f029d6242d1c